src/Microsoft.Identity.Client.Ref/Microsoft.Identity.Client.Ref.csproj

```diff
@@ -31,6 +31,12 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD_BUILDTIME</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
VS added this...should it be there? wasn't sure what to do.

Reply:
no, undo this file change.  VS is trying to be helpful and doesn't understand we have the msal.common.props file.